images/linux/scripts/base/apt-mock.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+prefix=/usr/local/bin
+
+for tool in apt apt-get apt-fast;do
+  real_tool=`which $tool`
+  cat >$prefix/$tool <<EOT
+#!/bin/sh
+
+i=1
+while [ \$i -le 10 ];do
+  $real_tool "\$@"
+  result=\$?
+  if [ \$result -eq  0 ];then
+    break
+  else
+    sleep 5
+    echo "...retry \$i"
```

Comment:
Do we want to add more logging here? What tool / command did fail?

Reply:
No, it is obvious from the previous message issued by the real tool.The apt is already verbose there's no need to add garbage